lib/matplotlib/testing/decorators.py

```diff
@@ -359,7 +360,7 @@ def check_figures_equal(*, extensions=("png", "pdf", "svg"), tol=0):
 
     Parameters
     ----------
-    extensions : list, default: ["png", "pdf", "svg"]
+    extensions : list, default: ["gif", "png", "pdf", "svg"]
```

Comment:
The default doesn't include `gif`.

Reply:
```suggestion
    extensions : list, default: ["png", "pdf", "svg"]
```

Reply:
Oops. I was toying with adding it as a default, but decided not to. Then I forgot to remove it from the docs. Fixed now.